EnglishFilter/sections/whitelist.txt

```diff
@@ -50,6 +50,9 @@ linkdrop.net$@$script[tag-content="absda"]
 inc42.com#@#.tag-social
 @@||piano.io^$domain=inc42.com
 @@||tinypass.com^$domain=inc42.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/72084
```

Comment:
I can reproduce with apps only. Why not `||tubewolf.com/_a_ta/s/s/$redirect=nooptext,important`?